infrastructure/views/api.py

```diff
@@ -163,7 +163,10 @@ def text_search(self, qs, text):
         if len(text) == 0:
             return qs
 
-        return qs.filter(content_search=SearchQuery(text))
+        result = qs.filter(
+            Q(project_description__contains=text) | Q(content_search=SearchQuery(text))
```

Comment:
Does it matter the search is case sensitive? I tried searching for `dist` as `P-CIN WAT DISTRIB POINT` appears in the test data but it returned no results. I then searched for `DIST` and it returned results.

Reply:
Changing the field lookup to `__icontains` seems to have solved case sensitivity for partial matches